### DISCUSSION TITLE
reload last file

### DISCUSSION THREAD

**Discussion by 20corvette** [arkypita/LaserGRBL#1963] [Q&A]
The reload last file key is no longer hot. Is there a setting to enable the key to go hot again?
Thank you

**Comment by arkypita** [comment]
Nothing was changed in LaserGRBL hotkey management.
Check your configuration (menu Grbl, Hotkeys)

**Comment by 20corvette** [comment]
From the config tab it appears as a hotkey but still is not functional?
[cid:81400968-0c28-45a4-aefc-02cfcd041ca2]
[cid:4dee69e4-9e00-482f-907b-699821f6ae26]The picture may not be clear but the reload last file is shaded
Thanks

________________________________

**Reply by 20corvette** [reply]
pics
![image](https://user-images.githubusercontent.com/119608921/205092555-732ae261-b315-410c-92df-361e5c19ecab.png)
![image](https://user-images.githubusercontent.com/119608921/205092775-01bb6014-ee4e-4f1c-9a43-5bd7338f829e.png)

**Reply by arkypita** [reply]
Reload last file is not available if you delete (move or rename) the last file you have opened, or if you never have opened a file, or if you are executing a job.

**Reply by 20corvette** [reply]
Yes...I understand and I have used the reload function several times in the past. However, nothing I do now will allow the reload to be hot. I am opening saved files from last projects, even "re-saving" them but the reload will still not function???

**Reply by arkypita** [reply]
This is how the software work.
The "Reload Last File" is disabled if not CanReopenFile.

CanReopenFile is implemented as I have described in previous message.
In my opinion it fails the "File.Exists" test.

![image](https://user-images.githubusercontent.com/8782035/205146974-cb27353f-a33c-45b5-800c-30796a4dc9c8.png)

**Comment by 20corvette** [comment]
Thank you.....

________________________________

**Reply by arkypita** [reply]
I'm sorry I can't help you more... if you find something let me know!

**Comment by 20corvette** [comment]
No worries, I appreciate your help. I deleted the hot key and reactivated…no change. I hard rebooted everything and no change. I created a new project (not from a previous saved project), processed it, saved it and now reload works. I don’t understand it but now it is working again. Go figure.
Thanks again for all your help and prompt replies.

Sent from my iPhone

On Dec 1, 2022, at 2:54 PM, arkypita ***@***.***> wrote:

﻿

I'm sorry I can't help you more... if you find something let me know!

**Comment by 20corvette** [comment]
I believe I may have found the issue. Did something change in the save project protocol?  I ran into other issues trying to modify past programs and saving the changes. Once I reopened all saved projects, saved them to a new folder using “quick save”….all appears to be working correctly. I will need to verify tomorrow but it appears to have resolved the issue.
Thanks again.

Sent from my iPhone

On Dec 1, 2022, at 3:26 PM, Mike Jaszewski ***@***.***> wrote:

﻿ No worries, I appreciate your help. I deleted the hot key and reactivated…no change. I hard rebooted everything and no change. I created a new project (not from a previous saved project), processed it, saved it and now reload works. I don’t understand it but now it is working again. Go figure.
Thanks again for all your help and prompt replies.

Sent from my iPhone

On Dec 1, 2022, at 2:54 PM, arkypita ***@***.***> wrote:

﻿

I'm sorry I can't help you more... if you find something let me know!
